packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.10.2",
+  "version": "6.11.0",
```

Comment:
Until now we did not release un-modified package

Reply:
Interesting, for x-license-pro, I observed a lot of published [packages](https://www.npmjs.com/package/@mui/x-license-pro?activeTab=versions) with no apparent changes, so I assumed we did it for this package for consistency. (For example 6.10.0 and 6.9.0 seem to have no related changes, yet the package is published)

By the way, doesn't it make sense to do it when doing minor semver update? To reduce the package version confusion for the new devs installing our packages?

Reply:
> (For example 6.10.0 and 6.9.0 seem to have no related changes, yet the package is published)

`6.10.0` did have a package-related change, while `6.9.0` does not seem to have one... We do sometimes release this package without a reason, but the consistency still leans towards releasing only when necessary. 🤔

Reply:
Sure. 👌 

Though I don't have a strong opinion on this, specifically for the `x-license-pro` package, I was leaning more towards syncing at least minor versions (if not patch) for the devs to have less confusion when installing a bunch of packages. But since we already have a convention, I suppose its better to keep it.

Reply:
> for the devs to have less confusion when installing a bunch of package

For me we either release every package or we only release the modified one.
Because if we do an exception for `x-license-pro`, it will still not be synchronized with all packages but only with the one released on the last release.
And this will happen more and more as we add smaller X packages.

____

Note that unless I'm mistaken, the core does not release non-modified package.